rlp/decode_test.go

```diff
@@ -437,6 +452,23 @@ var decodeTests = []decodeTest{
 		ptr:   new(recstruct),
 		error: "rlp: expected input string or byte for uint, decoding into (rlp.recstruct).Child.I",
 	},
+	{
+		input: "C0",
+		ptr:   new(invalidDotDot1),
+		error: "rlp: invalid struct tag \"..\" for rlp.invalidDotDot1.A (must be on last field)",
+	},
+	{
+		input: "C0",
+		ptr:   new(invalidDotDot2),
+		error: "rlp: invalid struct tag \"..\" for rlp.invalidDotDot2.B (field type is not slice)",
+	},
+
+	// struct tag ".."
+	{
+		input: "C3010203",
+		ptr:   new(dotdotRaw),
+		value: dotdotRaw{A: 1, DotDot: []RawValue{unhex("02"), unhex("03")}},
+	},
```

Comment:
Also it would be nice to see examples where:

 * `..` is used, but only one element is supplied
 * `..` is used, but 0 elements are supplied